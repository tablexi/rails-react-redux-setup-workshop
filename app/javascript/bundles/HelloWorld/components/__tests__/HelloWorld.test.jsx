import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HelloWorld from '../HelloWorld';

describe('<HelloWorld />', () => {
  it('renders', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(
      <HelloWorld name='Stranger' />
    );
    expect(tree).toMatchSnapshot();
  });
});
