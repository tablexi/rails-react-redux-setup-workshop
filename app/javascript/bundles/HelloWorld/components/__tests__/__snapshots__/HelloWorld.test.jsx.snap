// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloWorld /> renders 1`] = `
<div>
  <h3>
    Hello, 
    Stranger
    !
  </h3>
  <hr />
  <form>
    <label
      htmlFor="name"
    >
      Say hello to:
    </label>
    <input
      id="name"
      onChange={[Function]}
      type="text"
      value="Stranger"
    />
  </form>
</div>
`;
